Applications/Games/Teenagent/Online/script.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "GOG.Com",
```

Comment:
Would prefer only "GOG" (to make it more similar to "Steam" etc.). Or at least "GOG.com".

Reply:
Ok